### PR TITLE
docs: improve csv reader and monitor docs

### PR DESCRIPTION
**Pull request by bwanglzu**
documentation improvements in：

1. prepare training data section
2. run monitoring section

6F17A2D9-B362-48DC-A929-CF69638E10EE

92D70996-A4FF-44D2-90DB-E7129AA51FE0

---

- [ ] This PR references an open issue
- [x] I have added a line about this change to CHANGELOG

### CODE REVIEW

File: docs/walkthrough/create-training-data.md [hunk 1]
```diff
@@ -71,10 +87,22 @@ This is another English sentence    Dies ist ein weiterer englischer Satz
 When reading in files constructed in this way, a unique label is generated for each pair. Some sampling methods that require more than two elements per class will not work because each class in the dataset will only contain a single pair of elements.
 
 
+```
+
+```{warning}
+Please remove/replace comma in your data fields since csv use comma `,` as delimiter.
+
+
```
Comment (LMMilliken) [inline docs/walkthrough/create-training-data.md:95]:
```suggestion
```{warning}
Please remove/replace comma in your data fields if you are using a comma `,` as a delimiter.


```

File: docs/walkthrough/create-training-data.md [hunk 2]
```diff
@@ -71,10 +87,22 @@ This is another English sentence    Dies ist ein weiterer englischer Satz
...
 ```
 
 ````
 
+It is worth noting that:
+
++ Excel use `,` as delimiter and `\r\n` as lineterminator.
++ Excel-tab use `\t` as delimiter and `\r\n` as lineterminator.
++ Unix use `,` as delimiter and `\n` as lineterminator.
+
```
Comment (LMMilliken) [inline docs/walkthrough/create-training-data.md:105]:
```suggestion
It is worth noting that we support the following dialects:

+ `excel` use `,` as delimiter and `\r\n` as lineterminator.
+ `excel-tab` use `\t` as delimiter and `\r\n` as lineterminator.
+ `unix` use `,` as delimiter and `\n` as lineterminator.

```

**Review by LMMilliken — APPROVED** [review]
LGTM

File: docs/walkthrough/create-training-data.md [hunk 3]
```diff
@@ -35,7 +35,23 @@ https://example.com/apple-styling.jpg   picture of apple
 https://example.com/orange-styling.jpg  picture of orange
 ```
 
-If paths to local images are provided, they can be loaded into memory by setting `convert_to_blob = True` in the {class}`~finetuner.data.CSVOptions` object. It is worth noting that this setting does not cause Internet URLs to be loaded into memory.
+```diff
+import finetuner
+from finetuner import CSVOptions
```
Comment (gmastrapas) [inline docs/walkthrough/create-training-data.md:40]: is this importable at the top level?

**Review by gmastrapas — APPROVED** [review]

File: docs/walkthrough/create-training-data.md (hunk 3 above)
Comment (bwanglzu) [reply docs/walkthrough/create-training-data.md:40]: yes i runned in my tests

File: docs/walkthrough/create-training-data.md [hunk 4]
```diff
@@ -35,7 +35,23 @@ https://example.com/apple-styling.jpg   picture of apple
...
+    train_data='your-data.csv',
+-   csv_options=CSVOptions(),
++   csv_options=CSVOptions(is_labeled=True)
+)
+```
+
+```{important} 
+If paths to local images are provided,
+they can be loaded into memory by setting `convert_to_blob = True` in the {class}`~finetuner.data.CSVOptions` object.
+It is worth noting that this setting does not cause Internet URLs to be loaded into memory.
```
Comment (scott-martens) [inline docs/walkthrough/create-training-data.md:53]: "It is important to note that..."

"Worth noting" is too weak a wording for unexpected behaviour

File: docs/walkthrough/create-training-data.md [hunk 5]
```diff
@@ -68,13 +84,22 @@ This is another English sentence    Dies ist ein weiterer englischer Satz
...
-When reading in files constructed in this way, a unique label is generated for each pair. Some sampling methods that require more than two elements per class will not work because each class in the dataset will only contain a single pair of elements.
-
-
+When reading in files constructed in this way, a unique label is generated for each pair.
+Some sampling methods that require more than two elements per class will not work because each class in the dataset will only contain a single pair of elements.
 ```
 
 ````
 
+It is worth noting that we support the following dialects:
```
Comment (scott-martens) [inline docs/walkthrough/create-training-data.md:93]: "We support the following dialects of CSV:"

Direct language, and you should specify dialect of what since I don't see it from context.

File: docs/walkthrough/create-training-data.md [hunk 6]
```diff
@@ -68,13 +84,22 @@ This is another English sentence    Dies ist ein weiterer englischer Satz
...
+
++ `excel` use `,` as delimiter and `\r\n` as lineterminator.
++ `excel-tab` use `\t` as delimiter and `\r\n` as lineterminator.
++ `unix` use `,` as delimiter and `\n` as lineterminator.
+
+```{warning}
+Please remove/replace comma in your data fields if you are using a comma `,` as a delimiter.
```
Comment (scott-martens) [inline docs/walkthrough/create-training-data.md:100]: Does the quoting convention in CSV work in this importer?

"foo","bar","foo, bar"\n

Comment (bwanglzu) [reply docs/walkthrough/create-training-data.md:100]: if i understand you correctly, i think it will be imported as it is, including the quoting in column strings.
